Makefile

```diff
@@ -16,12 +16,10 @@ push:
 	bin/push.sh
 
 dev:
-	docker run								\
-	-it									\
-	--net=host								\
-	-v /vagrant/dcos-openvpn:/dcos						\
-	thomasr/dcos-openvpn							\
+	docker run -it --net=host       \
+	-v /vagrant/dcos-openvpn:/dcos  \
+	mesosphere/dcos-openvpn         \
 	/dcos/bin/test.sh
 
 setup-dns:
```

Comment:
Should probably just remove this, it was a half-thought solution to a problem that I'm not sure is a real problem (and it doesn't work at all right now).